docs/support/FAQ.md

```diff
@@ -0,0 +1,148 @@
+# Frequently Asked Questions
+
+## Why is the `package.json`’s version not updated in my repository?
+
+**semantic-release** takes care of updating the `package.json`’s version before publishing to [npm](https://www.npmjs.com).
+
+By default, only the published package will contains the version, which is the only place where it is *really* required, but the updated `package.json` will not be pushed to the Git repository
+
+However, the [`@semantic-release/git`](https://github.com/semantic-release/git) plugin can be used to push the updated `package.json` as well as other files to the Git repository.
+
+## How can I use a npm build script that requires the `package.json`’s version ?
+
+The `package.json`’s version will be updated by the `semantic-release` command just before publishing to [npm](https://www.npmjs.com), therefore it won't be available for scripts ran before the `semantic-release` command.
+
+As semantic-release uses the [npm CLI](https://docs.npmjs.com/cli/npm) to publish, all [npm hook scripts](https://docs.npmjs.com/misc/scripts#hook-scripts) will be executed. Therefore you can run your build script in the [`prepublishOnly`](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare) hook. It will be executed after the `semantic-release` command update the `package.json`’s version and before publishing.
+
+## Is there a way to preview which version would currently get published?
+
+Yes with the [dry-run options](../usage/configuration.md#dryrun) which prints to the console the next version to be published and the release notes.
+
+## Can I use semantic-release with Yarn?
+
+If you are using a [local](../usage/installation.md#local-installation) **semantic-release** installation and run multiple CI jobs with different versions, the `yarn install` command will fail with Node < 8 as **semantic-release** require [Node >= 8](#why-does-semantic-release-require-node-version--8). See [yarnpkg/rfcs#69](https://github.com/yarnpkg/rfcs/pull/69).
+
+In order to run **semantic-release** with [Yarn](https://yarnpkg.com) you would need to:
+- Use a [global](../usage/installation.md#global-installation) **semantic-release** installation
+- Make sure to install and run the `semantic-release` command only in a CI jobs running with [Node >= 8](#why-does-semantic-release-require-node-version--8)
```

Comment:
Wouldn't it be much easier to just use `--ignore-engines`? That makes it behave like npm (only print warnings)

Reply:
This is what we suggested when the problem was reported, but for some reason it wasn't working due to a bug in `yarn` that was somehow ignoring `--ignore-engines`.

If it's fixed the doc can be changed